masterbase/app.py

```diff
@@ -140,6 +141,15 @@ async def demodata(request: Request, api_key: str, session_id: str) -> Stream:
     return Stream(bytestream_generator, media_type=MediaType.TEXT, headers=headers)
 
 
+@post("/report", guards=[valid_key_guard])
+async def report_player(request: Request, api_key: str, session_id: str, target_steam_id: str):
+    """Add a player report."""
```

Comment:
This endpoint also needs to be added to this list: https://github.com/MegaAntiCheat/masterbase/blob/main/masterbase/app.py#L319

Reply:
and a return type needs to be added of dict[str, bool]

Reply:
added type annotations + populated the list (https://github.com/MegaAntiCheat/masterbase/pull/55/commits/cbe98ad8a883be910aa6d2cb14a4d23b1ae33471)